Formula/temporalite.rb

```diff
@@ -0,0 +1,25 @@
+class Temporalite < Formula
+  desc "Experimental distribution of Temporal that runs as a single process"
+  homepage "https://github.com/temporalio/temporalite"
+  url "https://github.com/temporalio/temporalite/archive/v0.3.0.tar.gz"
+  sha256 "a4980cca9b265694dca97e696fbe9136b2bb4ceecfc44a43cfc7d1ac55a3009b"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "tctl" => :test
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/temporalite"
+  end
+
+  test do
+    fork do
+      exec "temporalite start --headless --ephemeral --namespace default"
+    end
+
+    sleep 5
+
+    system "tctl", "namespace", "list"
+    system "tctl", "workflow", "list"
```

Comment:
Is there some output we could check here?

Reply:
➕

Reply:
This must be possible. Here is the output from `tctl namespace list` which shows that the `temporal-system` and `default` namespaces were created on startup.

How do you prefer to test the output of these kinds of commands? Can we pipe the output into grep and check that the line `Name: default` exists in the output?

```
$ tctl namespace list
Name: temporal-system
Id: 32049b68-7872-4094-8e63-d0dd59896a83
Description: Temporal internal system namespace
OwnerEmail: temporal-core@temporal.io
NamespaceData: map[string]string(nil)
State: Registered
Retention: 168h0m0s
ActiveClusterName: active
Clusters: active
HistoryArchivalState: Disabled
IsGlobalNamespace: false
FailoverVersion: 0
FailoverHistory: []
VisibilityArchivalState: Disabled
Bad binaries to reset:
+-----------------+----------+------------+--------+
| [94mBINARY CHECKSUM[0m | [94mOPERATOR[0m | [94mSTART TIME[0m | [94mREASON[0m |
+-----------------+----------+------------+--------+
+-----------------+----------+------------+--------+
Name: default
Id: ac7f926f-0bce-41a4-931e-3e2e98a36c32
Description: 
OwnerEmail: 
NamespaceData: map[string]string(nil)
State: Registered
Retention: 24h0m0s
ActiveClusterName: active
Clusters: active
HistoryArchivalState: Disabled
IsGlobalNamespace: false
FailoverVersion: 0
FailoverHistory: []
VisibilityArchivalState: Disabled
Bad binaries to reset:
+-----------------+----------+------------+--------+
| [94mBINARY CHECKSUM[0m | [94mOPERATOR[0m | [94mSTART TIME[0m | [94mREASON[0m |
+-----------------+----------+------------+--------+
+-----------------+----------+------------+--------+
```

Reply:
`assert_match "expected", shell_output("command")`